buildSrc/src/main/groovy/com/google/firebase/gradle/GenerateMeasurementsTask.groovy

```diff
@@ -75,52 +76,82 @@ public class GenerateMeasurementsTask extends DefaultTask {
 
     @TaskAction
     def generate() {
-        def sdks = createSdkMap()
-        def sizes = calculateSizes(sdks, project.android.applicationVariants)
-        def report = createReport(sizes)
+        // Check if we need to run human-readable or upload mode.
+	if (project.hasProperty("pull_request")) {
+            def pullRequestNumber = project.properties["pull_request"]
```

Comment:
Formatting wierdness

Reply:
Fixed. My editor seems to be throwing tabs in... :(